ui/app/components/namespace-picker.hbs

```diff
@@ -4,20 +4,32 @@ SPDX-License-Identifier: BUSL-1.1
 }}
 
 <div class="namespace-picker" ...attributes>
-  <Hds::Form::SuperSelect::Single::Field
-    @afterOptionsComponent={{component "namespace-picker/after-options" loadOptions=this.loadOptions}}
-    @ariaLabel="Namespace"
-    @onChange={{this.onChange}}
-    @options={{this.options}}
-    @placeholder="Search"
-    @searchEnabled={{true}}
-    @selected={{this.selected}}
-    @selectedItemComponent={{component "namespace-picker/selected-option"}}
-    @showAfterOptions={{this.showAfterOptions}}
-    @verticalPosition="above"
-    data-test-namespace-toggle
-    as |F|
-  >
-    <F.Options>{{F.options.label}}</F.Options>
-  </Hds::Form::SuperSelect::Single::Field>
+  <Hds::Dropdown @enableCollisionDetection={{true}} as |D|>
+
+    <D.ToggleButton @icon="org" @text={{this.selectedNamespace}} data-test-namespace-toggle />
+
+    <D.Title @text={{component "namespace-picker/title" count=this.options.length}} />
```

Comment:
Just out of curiosity was using ` <D.Generic>` weird stylistically? It seems a little more straightforward to render the title/count directly instead of passing it as a `@text` arg (which wow - I'm impressed you figured that out! 😂 🙌  )

Especially since the title component is so simple separating it out seems a little verbose and keeping it all here would read a little easier.

Reply:
I'll try out `<D.Generic>` and see - I was going based off of the HDS code example, which used `Title` but `Generic` is probably more straightforward if it allows me to get rid of the component. Thanks for calling this out!!